### PR TITLE
Container Images used report

### PR DESCRIPTION
**Pull request by rporres**
```
$ qontract-cli --config config.development.toml get  container-images --help
Usage: qontract-cli get container-images [OPTIONS]

  Get all container images in app-interface defined namespaces

Options:
  --cluster-name TEXT             openshift cluster names to act on i.e.:
                                  --cluster-name cluster-1 --cluster-name
                                  cluster-2
  --namespace-name TEXT           namespace name to act on.
  --thread-pool-size INTEGER      number of threads to run in parallel.
  --use-jump-host / --no-use-jump-host
                                  use jump host if defined.
  --exclude-pattern TEXT          Exclude images that match this pattern
  --include-pattern TEXT          Only include images that match this pattern
  --help                          Show this message and exit.
```

APPSRE-11229

### CODE REVIEW

**Comment by rporres** [comment]
example

```
$ qontract-cli --config config.development.toml get container-images \
   --cluster-name app-sre-stage-01 
   --exclude-pattern 'quay.io/redhat-services-prod|quay.io/redhat-user-workloads|registry.redhat.io'  \
   --namespace-name uhc-integration
NAME                                  NAMESPACES         COUNT
------------------------------------  ---------------  -------
quay.io/app-sre/envoyproxy            uhc-integration        9
quay.io/app-sre/uhc-clusters-cleaner  uhc-integration        6
quay.io/app-sre/ocm-envoy             uhc-integration       18
quay.io/app-sre/uhc-clusters-service  uhc-integration        8
quay.io/openshift/origin-oauth-proxy  uhc-integration        1
quay.io/app-sre/gabi                  uhc-integration        1
quay.io/app-sre/uhc-gateway           uhc-integration        3
quay.io/app-sre/ocm-addons-service    uhc-integration        6
quay.io/app-sre/uhc-account-manager   uhc-integration       34
quay.io/app-sre/ocm-backend-tests     uhc-integration       10
quay.io/app-sre/ms-interop-tests      uhc-integration        1
quay.io/interop_qe_ms/alpine-utils    uhc-integration        1
quay.io/app-sre/ocm-api-tests         uhc-integration        3
quay.io/app-sre/ocm-managed-services  uhc-integration        6
quay.io/app-sre/ocm-service-log       uhc-integration        6
```

File: tools/test/test_get_container_images.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import pytest
+from pytest_mock import MockerFixture
+
+from reconcile.gql_definitions.common.namespaces_minimal import ClusterV1, NamespaceV1
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.test.fixtures import Fixtures
+from reconcile.utils.oc import OCNative
+from reconcile.utils.oc_map import OCMap
+from tools.cli_commands.container_images_report import (
+    _fetch_pods_images_from_namespaces,
```
Comment (hemslo) [inline tools/test/test_get_container_images.py:10]: can remove `_` prefix in function name as this is the main tested public one.

**Review by hemslo — APPROVED** [review]
LGTM!

File: tools/test/test_get_container_images.py (hunk 1 above)
Comment (rporres) [reply tools/test/test_get_container_images.py]: sure, I was hesitant since I only created that function for testing, but it makes sense.

**Pull request merged**